src/main/java/org/umssdiplo/automationv01/core/managepage/home/SSIAHome.java

```diff
@@ -36,4 +43,10 @@ public RoleList clickOnRoleMenu() {
         CommonEvents.clickButton(roleMenu);
         return new RoleList();
     }
+
+    public AuditList clickAuditMenu() {
+        CommonEvents.clickButton(auditsMenu);
```

Comment:
it will be great if your next pulls request you can add a page object instead of this clickButton.